README.md

```diff
@@ -127,3 +127,8 @@ The following are `gitmoji` recommendations for the `scope`. These are not curre
 - Deps - :lock: `:lock:`
 - Breaking changes - :boom: `:boom:`
 - Config - :wrench: `:wrench:`
+
+### Troubleshooting
+
+- Error `**.sh: not found` when running skaffold in Windows.
```

Comment:
In my experience, Windows developers usually will set up git to convert back and forth between line ending types automatically on push/pull